example_data/sample_bundle.json

```diff
@@ -470,34 +533,77 @@
             ]
           }
         ],
-        "valueString": "Next generation sequence analysis indicates that General Manager is heterozygous for the MECP2 c.953G>C p.(Arg318Pro) likely pathogenic variant that has been confirmed by Sanger sequence analysis"
+        "effectiveDateTime": "2020-10-22",
```

Comment:
the effective time here is new, what date is this representing?

Reply:
This was a massive error. We used it to represent the authorized datetime and I forgot to remove it. Updated now